addons/pos_self_order/static/src/kiosk/pages/order_cart/order_cart.js

```diff
@@ -0,0 +1,79 @@
+/** @odoo-module */
+
+import { Component } from "@odoo/owl";
+import { useService } from "@web/core/utils/hooks";
+import { ProductCard } from "@pos_self_order/kiosk/components/product_card/product_card";
+import { useselfOrder } from "@pos_self_order/kiosk/self_order_kiosk_service";
+import { KioskTemplate } from "@pos_self_order/kiosk/template/kiosk_template";
+
+export class OrderCart extends Component {
+    static template = "pos_self_order.OrderCart";
+    static components = { ProductCard, KioskTemplate };
+
+    setup() {
+        this.selfOrder = useselfOrder();
+        this.router = useService("router");
+    }
+
+    get lines() {
+        const lines = this.selfOrder.currentOrder.lines;
+        return lines ? lines : [];
+    }
+
+    backToMenu() {
+        this.router.navigate("product_list");
+    }
+
+    pay() {
+        if (this.selfOrder.kiosk_mode === "table") {
+            this.router.navigate("table_number");
+        } else {
+            this.router.navigate("payment");
+        }
+    }
+
+    async removeLine(lineUuid) {
+        this.selfOrder.currentOrder.removeLine(lineUuid);
+        await this.selfOrder.getPricesFromServer();
+    }
+
+    getChildLines(line) {
+        const children = [];
+        for (const cline of this.lines) {
+            if (cline.combo_parent_id === line.uuid) {
+                children.push(cline);
+            }
+        }
+        return children;
+    }
```

Comment:
return this.lines.filter((l) => l.combo_parent_id === line.uuid)
```suggestion
    getChildLines(line) {
       return this.lines.filter((l) => l.combo_parent_id === line.uuid)
    }
```